content/docs/reference-react-component.md

```diff
@@ -250,6 +256,20 @@ componentWillUnmount()
 
 * * *
 
+### `componentDidCatch()`
+
+```javascript
+componentDidCatch()
+```
+
+This is a new feature in React 16 to improve error handling on UI crashes.
+
+A React class component becomes an "error boundary" if this method is declared in it. Generally, it catches JavaScript errors when the rendering of its **children** crashes, logs the error and displays a fallback UI.
+
+For more details, check [*Error Handling in React 16*](/blog/2017/07/26/error-handling-in-react-16.html).
```

Comment:
What do you think of this wording? (Dropping the "this is a new feature" paragraph and then...)

> Error boundaries are React components that **catch JavaScript errors anywhere in their child component tree, log those errors, and display a fallback UI** instead of the component tree that crashed. Error boundaries catch errors during rendering, in lifecycle methods, and in constructors of the whole tree below them.
>
> A class component becomes an error boundary if it defines this lifecycle method.
>
> For more details, see [*Error Handling in React 16*](/blog/2017/07/26/error-handling-in-react-16.html).
>
> **Note**
> Error boundaries only catch errors in the components below them in the tree. An error boundary can’t catch an error within itself.